### PR TITLE
New multicurrency module

### PR DESCRIPTION
**Pull request by aconits**
Add functionnality with multicurrency module. Can choose default currency on company card then create propale and add new line

### CODE REVIEW

**Comment by aspangaro** [comment]
:+1:

**Comment by defrance** [comment]
love it :heart_eyes:

**Comment by aconits** [comment]
Conflicts solved, ready to merge @eldy  :smile:

File: htdocs/admin/multicurrency.php [hunk 1]
```diff
@@ -0,0 +1,289 @@
...
+ * 	\ingroup	quickpriceupdate
+ * 	\brief		This file is an example module setup page
+ * 				Put some comments here
+ */
+// Dolibarr environment
+
+require '../main.inc.php';
+
+// Libraries
+dol_include_once('/core/lib/admin.lib.php');
```
Comment (jfefe) [inline htdocs/admin/multicurrency.php:30]: This function should be used by external module, not for a core module. 
For core module, please use require_once '...' to limit performance issue.

Comment (aconits) [inline htdocs/admin/multicurrency.php:30]: :+1:  All right, if this feature is merge I'll work again to fix it.  Actually i'm waiting for good return about this PR to integrate the rest.

**Comment by altairis-tof** [comment]
hope this PR will be merged soon so you can merge the rest; thanks for this great improvement for Dolibarr next major release !

**Comment by simnandez** [comment]
manages the module the conversion rate if a customer pays in dollars and our bank is in euros, (e.g.)?

**Comment by aconits** [comment]
It's one goal of this module

**Comment by tiaris** [comment]
Thank you for this new function

**Comment by aspangaro** [comment]
:+1:

**Pull request merged**